v2/event.go

```diff
@@ -142,7 +142,8 @@ func newEvent(rawData []interface{}, notifier *Notifier) (*Event, *Configuration
 			event.Stacktrace = make([]StackFrame, len(err.StackFrames()))
 
 		case bool:
-			config = config.merge(&Configuration{Synchronous: bool(datum)})
+			config = config.clone()
```

Comment:
OK - I see. Maybe just a comment to capture this and stop a well-meaning refactor breaking it?

```suggestion
			// clone without merging to avoid double-call to Configuration.update
			config = config.clone()
```